### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by amanjeev**
- Englishize the error messages in major errors.rs.
- Elide lifetimes where applicable.

### CODE REVIEW

**Comment by amanjeev** [comment]
So strange, the tests pass locally. :sob:. Will see this tomorrow.

**Comment by amanjeev** [comment]
Wow! I am an idiot.

**Comment by amanjeev** [comment]
Phew! That was close.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Standardize error messages as close to English rules as possible.
```
Comment (Hoverbear) [inline CHANGELOG.md:12]: Other entries below use past-tense. 
```suggestion
- Standardized error messages as close to English rules as possible.
```

File: crates/criticalup-cli/tests/snapshots/cli__auth_set__byte_zero_via_stdin.snap [hunk 2]
```diff
@@ -1,15 +1,12 @@
 ---
 source: crates/criticalup-cli/tests/cli/auth_set.rs
-assertion_line: 22
```
Comment (Hoverbear) [inline crates/criticalup-cli/tests/snapshots/cli__auth_set__byte_zero_via_stdin.snap:3]: Why did all these disappear?

**Pull request merged**